Resources/public/js/views/serverside/ez-roleserversideview.js

```diff
@@ -82,5 +84,23 @@ YUI.add('ez-roleserversideview', function (Y) {
         _uiUnsetAssignRoleLoading: function (button) {
             button.removeClass('is-loading').set('disabled', false);
         },
+
+        /**
+         * Refreshes the role view, showing the assignments tab.
+         *
+         * @method _refreshAssignmentsTab
+         * @protected
+         * @param {Y.Node} button
+         */
+        _refreshAssignmentsTab: function (button) {
+            /**
+             * Fired when the view needs to be refreshed.
+             * @event refreshAssignmentsTab
+             * @param roleId {String} The ID of the role we are showing assignments for
+             */
+            this.fire('refreshAssignmentsTab', {
```

Comment:
in https://github.com/ezsystems/PlatformUIBundle/commit/f4b909cdedbc01ea848c6058111a574a4c518799, @yannickroger did pretty much the same for section view, I guess we need to move what he did to the ServerSideViewService and ServerSideView instead of duplicating the code.